debian/rules

```diff
@@ -19,11 +19,12 @@ export DEB_HOST_ARCH ?= $(shell dpkg-architecture -qDEB_HOST_ARCH)
 %:
 	dh $@ --buildsystem=cmake --builddirectory=build --parallel
 
-export ROS_VERSION ?= kinetic
-export ROS_BASE_PREFIX ?= /opt/ros
-export ROS_INSTALL_PREFIX ?= /opt/ros
-export NNS_INSTALL_PREFIX ?= /usr
-export NNS_INSTALL_LIBDIR ?= lib/${DEB_HOST_MULTIARCH}
+export ROS_VERSION = kinetic
```

Comment:
Does it work?
```
$ export A = b
bash: export: `=': not a valid identifier
$ export A=b
$ 
```

Reply:
Yes, it works. 
```bash
dh_auto_configure -- -DROS_VERSION=kinetic \
	-DROS_BASE_PREFIX=/opt/ros \
	-DROS_INSTALL_PREFIX=/opt/ros \
	-DNNS_INSTALL_PREFIX=/usr \
	-DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu \
	-DCMAKE_LIBRARY_ARCHITECTURE="x86_64-linux-gnu"
	mkdir -p build
	cd build
	cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DROS_VERSION=kinetic -DROS_BASE_PREFIX=/opt/ros -DROS_INSTALL_PREFIX=/opt/ros -DNNS_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu -DCMAKE_LIBRARY_ARCHITECTURE=x86_64-linux-gnu
```

FYI, auto-generated comments for guiding debian packaging also use those grammar as follows (I do not know why) 
```bash
# see FEATURE AREAS in dpkg-buildflags(1)
#export DEB_BUILD_MAINT_OPTIONS = hardening=+all

# see ENVIRONMENT in dpkg-buildflags(1)
# package maintainers to append CFLAGS
#export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
# package maintainers to append LDFLAGS
#export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
```